### PR TITLE
[change] Clarified captive_portal_login_form and captive_portal_logout_form #782

### LINKED ISSUE #782: [docs] Clarify captive_portal_login_form and captive_portal_logout_form
The README doesn't explain these critical configuration sections in detail, we must add a section that explains these.

### PR DESCRIPTION
**Pull request by praptisharma28**
Fixes #782

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -907,6 +907,61 @@ should point to the mock app just created. This way captive portal login
 and logout requests will not hang, allowing users to view/modify their
 account data also from the public internet.
 
+### `captive_portal_login_form` :
+
+This section defines the configuration for the captive portal login form. The application uses an HTTP `POST` request to the `action` URL. The request body is structured as defined by the `fields` property.
```
Comment (nemesifier) [inline README.md:912]: make sure line length is wrapped around 75 characters.

File: README.md [hunk 2]
```diff
@@ -907,6 +907,61 @@ should point to the mock app just created. This way captive portal login
...
+
+#### Explanation:
+
+- The method is set to post for HTTP POST request.
+- The action is set to `http://localhost:8000/captive-portal-mock/login/` which should be replaced with the actual captive portal login URL.
+- The fields dictionary maps form fields username and password to the parameters `auth_user` and `auth_pass expected` by the captive portal.
+- `macaddr_param_name` is set to macaddr which means the MAC address is passed in a parameter with the name macaddr.
+- The `additional_fields` array defines extra parameters for the login request.
+
+#### Note: The values within the additional_fields array should be modified based on your captive portal's specific requirements. The values in the example above are just a starting point.
```
Comment (nemesifier) [inline README.md:939]: Don't use headings for notes please.

File: browser-test/initialize_data.py [hunk 3]
```diff
@@ -101,9 +99,6 @@ def load_test_data():
...
         password=data['password'],
         email=data['email'],
-        password_updated=now().date()-timedelta(days=180)
-    )
-    OrganizationUser.objects.create(
-        organization=org,
-        user=expired_password_user
+        password_updated=now().date() - timedelta(days=180),
     )
+    OrganizationUser.objects.create(organization=org, user=expired_password_user)
```
Comment (nemesifier) [inline browser-test/initialize_data.py:104]: These code changes are not relevant in this docs PR, please revert these, focus each PR on one goal.

**Review by nemesifier — CHANGES_REQUESTED** [review]
Thanks @praptisharma28, please see my comments below.
Have you already tried running those texts to AI tools to improve them?
ChatGPT and Google Gemini are pretty good at this.

File: README.md [hunk 4]
```diff
@@ -907,6 +907,61 @@ should point to the mock app just created. This way captive portal login
 and logout requests will not hang, allowing users to view/modify their
 account data also from the public internet.
 
+### `captive_portal_login_form` :
```
[the remark this answers is not in the document]
Comment (praptisharma28) [reply README.md:910]: Ok @nemesifier can you tell me the exact location because I tried too look for a it earlier too, I went with where it's mention is.

**Comment by praptisharma28** [comment]
> Thanks @praptisharma28, please see my comments below. Have you already tried running those texts to AI tools to improve them? ChatGPT and Google Gemini are pretty good at this.

Okay I will take AI help and will improve.

File: README.md (hunk 2 above)
Comment (praptisharma28) [reply README.md:939]: Okay

File: browser-test/initialize_data.py (hunk 3 above)
Comment (praptisharma28) [reply browser-test/initialize_data.py:104]: @nemesifier , I ran ```openwisp-qa-format``` and it did this. But I will revert before pushing, np.

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (nemesifier) [reply README.md:910]: Let's move it to the first positions under "Settings".

[the remark this answers is not in the document]
Comment (praptisharma28) [reply README.md:910]: ok Federico

**Review by nemesifier — COMMENTED** [review]
Thanks @praptisharma28, I used this as a base for https://github.com/openwisp/openwisp-wifi-login-pages/pull/785.

**Pull request merged**